README.rst

```diff
@@ -225,13 +225,13 @@ be deleted.
 .. raw:: html
 
   <embed><table>
-    <tr> <th>Title</th> </tr>
-    <tr> <td class="left"><a href="zips/draft-arya-dairaemma-disable-addition-of-transparent-chain-value.md">Disabling Addition of New Value to the Transparent Chain Value Pool</a></td>
-    <tr> <td class="left"><a href="zips/draft-arya-deploy-nu7.md">Deployment of the NU7 Network Upgrade</a></td>
-    <tr> <td class="left"><a href="zips/draft-ecc-authenticated-reply-addrs.md">Authenticated Reply Addresses</a></td>
-    <tr> <td class="left"><a href="zips/draft-ecc-onchain-accountable-voting.md">On-chain Accountable Voting</a></td>
-    <tr> <td class="left"><a href="zips/draft-ecc-quantum-recoverability.md">Quantum recoverability</a></td>
-    <tr> <td class="left"><a href="zips/draft-str4d-orchard-balance-proof.md">Air drops, Proof-of-Balance, and Stake-weighted Polling</a></td>
+    <tr> <th>Draft name</th> <th>Title</th> <th>Discussions-To</th> </tr>
+    <tr> <td class="left">draft-arya-dairaemma-disable-addition-of-transparent-chain-value</td> <td class="left"><a href="zips/draft-arya-dairaemma-disable-addition-of-transparent-chain-value.md">Disabling Addition of New Value to the Transparent Chain Value Pool</a></td> <td class="left"><a href="https://github.com/zcash/zips/issues/1115">zips#1115</a></td>
```

Comment:
I don't think the "Draft name" column adds much value, as it is mostly duplicative of the existing "Title" column (unlike the "ZIP" column in the earlier tables). But I won't object to trying it; we can always remove it later.

Reply:
I like to be able to remember the filenames, and this helps a little with that.